### PR TITLE
[#166816802] Add required_channel_privacy_level field to NewMessage definition

### PR DESCRIPTION
**Pull request by francescopersico**
This PR is required for another almost finished PR in the io-functions-services project.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#166816802](https://www.pivotaltracker.com/story/show/166816802): Aggiungere metadato al servizio che inibisce l'inoltro via mail dei messaggi inviati da un determinato servizio

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ff8fbb6d9912167ab72b8d4175eb51e7716b0b11
</p>

File: openapi/definitions.yaml [hunk 1]
```diff
@@ -102,6 +102,8 @@ NewMessage:
       $ref: "#/MessageContent"
     default_addresses:
       $ref: "#/NewMessageDefaultAddresses"
+    required_channel_privacy_level:
```
Comment (cloudify) [inline openapi/definitions.yaml]: since we have only two privacy levels, why don't we start with a simple boolean?

Comment (cloudify) [reply openapi/definitions.yaml]: @francescopersico let's discuss about this

Comment (francescopersico) [reply openapi/definitions.yaml]: We decided to set the privacy at Service level. I already have a local branch with a field called "requireSecureChannel" defined at Service level (option boolean with default of FALSE). I am going to close this PR and open a new one that implements the behaviour decided on PivotalTracker.

**Comment by francescopersico** [comment]
Implemented at service level in #28

**Pull request closed without merging**